lib/methods/pluginchecktask.cpp

```diff
@@ -66,15 +67,22 @@ void PluginCheckTask::ProcessFinishedHandler(const Checkable::Ptr& checkable, co
 	Checkable::CurrentConcurrentChecks.fetch_sub(1);
 	Checkable::DecreasePendingChecks();
 
+	String output = pr.Output.Trim();
+
 	if (pr.ExitStatus > 3) {
 		Process::Arguments parguments = Process::PrepareCommand(commandLine);
 		Log(LogWarning, "PluginCheckTask")
 			<< "Check command for object '" << checkable->GetName() << "' (PID: " << pr.PID
 			<< ", arguments: " << Process::PrettyPrintArguments(parguments) << ") terminated with exit code "
 			<< pr.ExitStatus << ", output: " << pr.Output;
-	}
 
-	String output = pr.Output.Trim();
+		std::stringstream crOutput;
+
+		crOutput << "<Terminated with exit code " << pr.ExitStatus
+			<< " (0x" << std::noshowbase << std::hex << std::uppercase << pr.ExitStatus << ").>";
```

Comment:
> Notice also that no base prefix is implicitly prepended to the number unless the [showbase](https://cplusplus.com/showbase) format flag is set.

[`std::hex`](https://cplusplus.com/reference/ios/hex/) doesn't prepend the base unless you explicitly request it to.

Reply:
This is nice, but obvious code is even nicer.